packages/components/src/dropdown-menu/index.tsx

```diff
@@ -120,6 +132,9 @@ function UnconnectedDropdownMenu( dropdownMenuProps: DropdownMenuProps ) {
 				return (
 					<Toggle
 						{ ...mergedToggleProps }
+						{ ...( toggleProps?.as === undefined
+							? { __shouldNotWarnDeprecated36pxSize: true }
```

Comment:
This is to prevent a redundant warning from `Button` when `DropdownMenu` has already warned.